apps/src/storage/firebaseUtils.js

```diff
@@ -1,9 +1,12 @@
-/* global Applab */
-
 import Firebase from 'firebase';
+let config;
 let firebaseCache;
 let firebaseConfig;
 
+export function init(setupConfig) {
+  config = setupConfig;
+}
```

Comment:
I would recommend verifying that the config contains all the necessary fields here. It probably won't make a difference for the stuff used by getDatabase, but a missing `showRateLimitAlert` could go unnoticed for quite some time (not sure we have tests covering that particular error)